userguide/jails.rst

```diff
@@ -1233,14 +1192,14 @@ shows the :guilabel:`Configuration Options` for audiotag.
 This port has five configurable options (DOCS, FLAC, ID3, MP4,
 and VORBIS) and each option is enabled (on) by default.
 
-FreeBSD packages are always built using the default options. When
-compiling a port, those options are presented in a menu, allowing the
-default values to be changed.
+FreeBSD packages are always built using the default options. Those
```

Comment:
This is confusingly said.  Packages use default options.  Ports let the user select options.